### PR TITLE
fix: Entry is named

### PR DESCRIPTION
**Pull request by nlewis84**
- Allows for easier overriding of text on Auth Entry page.

### CODE REVIEW

File: packages/apollos-ui-auth/src/Entry.js [hunk 1]
```diff
@@ -50,12 +51,12 @@ const Entry = ({
   promptText,
   setFieldValue,
   tabTitle,
-  theme,
   values,
   inputAutoComplete,
   alternateLogin,
 }) => {
   const handleOnChangeText = (text) => setFieldValue(inputType, text);
+  const source = useTheme();
```
Comment (redreceipt) [inline packages/apollos-ui-auth/src/Entry.js:59]:
```suggestion
  const theme = useTheme();
```

**Comment by redreceipt** [comment]
cc @conrad-vanl hold up a min :)

**Comment by redreceipt** [comment]
@nlewis84 add "fix" to the title and close and reopen the PR

**Comment by redreceipt** [comment]
this will probably change snaps a bit but that's ok, I'll show you how to update when if that happens

**Comment by redreceipt** [comment]
great work btw!!

File: packages/apollos-ui-auth/src/Entry.js [hunk 2]
```diff
@@ -101,7 +102,7 @@ const Entry = ({
                       enablesReturnKeyAutomatically
                       error={get(errors, inputType)}
                       label={inputLabel}
-                      labelColor={theme.colors.text.tertiary}
+                      labelColor={source.colors.primary}
```
Comment (redreceipt) [inline packages/apollos-ui-auth/src/Entry.js:105]: is there a reason you changed this to primary? it should probably stay the same

Comment (nlewis84) [reply packages/apollos-ui-auth/src/Entry.js:105]: Yeah, when I logged 'source' I got this:

`fellowshipNWA: [Tue Jul 27 2021 09:45:40.156]  LOG      {"colors": {"background": "#F8F8FB", "border": "rgb(216, 216, 216)", "card": "#FFFFFF", "notification": "rgb(255, 59, 48)", "primary": "#4E9AF4", "text": "#27272E"}, "dark": false}`

Comment (nlewis84) [reply packages/apollos-ui-auth/src/Entry.js:105]: Could have been either source.colors.primary or source.colors.text.

Comment (redreceipt) [reply packages/apollos-ui-auth/src/Entry.js:105]: theme is messed up on fellowship. do your dev work on core to eliminate confusion. run `yarn app` with your branch or pull up the component in storybook.

Comment (nlewis84) [reply packages/apollos-ui-auth/src/Entry.js:105]: Stuck on this one for the time being, until I can get the core-app running. Still getting no bundle URL present.

File: packages/apollos-ui-auth/src/Entry.js (hunk 1 above)
Comment (nlewis84) [reply packages/apollos-ui-auth/src/Entry.js]: got rid of useTheme()

**Comment by nlewis84** [comment]
@conrad-vanl Running tests on this. I had trouble isolating this and only making the Entry named. I styled and named the TextInput as well. Testing overriding on ui-auth.Entry and ui-auth.StyledTextInput and both worked as expected.

**Review by redreceipt — APPROVED** [review]
Nice!

**Comment by nlewis84** [comment]
@redreceipt There's a postgres test that is failing, but unrelated to this PR. What should I do here?

**Pull request merged**